### PR TITLE
[show command] allow warm restart table contains empty restore_count

### PR DESCRIPTION
**Pull request by yxieca**
Signed-off-by: Ying Xie <ying.xie@microsoft.com>

**- What I did**
In practice, a software bug could cause not having restore_count, and
we could have use case (e.g. warm-shutdown) has state without
restore_count.

It would be nice that the command show all these rows without exception.

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -1624,7 +1624,10 @@ def remove_prefix(text, prefix):
         entry = db.get_all(db.STATE_DB, tk)
         r = []
         r.append(remove_prefix(tk, prefix))
-        r.append(entry['restore_count'])
+        if 'restore_count' not in  entry:
```
Comment (stcheng) [inline show/main.py]: extra space before entry

Comment (yxieca) [reply show/main.py]: Copied from next if block :-) Will remove it.

Comment (yxieca) [reply show/main.py]: Done. Removed both.

**Comment by yxieca** [comment]
yes. 2 stage warm shutdown.

I want to reuse the existing show command. But don't really want having to add an artificial restore_count. I use this table to sync up warm-reboot script and syncd pre-shutdown state.

**Pull request merged**